### ISSUE TITLE
Feature request: A field for interface

### ISSUE THREAD

**Issue by Arendtsen** [SpriteLink/NIPAP#735]
A field for interface name.
This could be on a router or on a host.

**Comment by plajjan** [comment]
This is one I have been pondering for a long time. It could be quite handy to store interface information but I don't know how to store it. Is a simple text string enough? Should there be more intelligence behind it, potentially splitting interface type (TenGigabitEth) and id (0/0/0/1)?

These days we have AVPs/extra-attributes, so you could actually use that to store your interface information, at least in the meantime.

**Comment by toreanderson** [comment]
I'd just like to note that for HA setups using VRRP or similar, you might have multiple interfaces on multiple routers.

**Comment by plajjan** [comment]
Indeed, the same issue is present for nodes though and we have an attribute for that so I don't think it's much for an issue with introducing another field. If you do hsrp you simply need to handle it some other way.
